ProjFS.Mac/PrjFSKext/public/Message.h

```diff
@@ -32,6 +32,14 @@ typedef enum
     
 } MessageType;
 
+enum MessagePathField
+{
+    MessagePath_Target = 0,
+    MessagePath_From,
+    
+    MessagePath_Count,
+};
```

Comment:
So this is the second option for removing the `union` from the original approach - stick with the array and name each element in the array using this `enum`.